android/build.gradle

```diff
@@ -1,11 +1,21 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.3'
+    }
+}
+
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion rootProject.ext.compileSdkVersion
+    compileSdkVersion 30
```

Comment:
Why did we remove rootProject.ext.compileSdkVersion instead of just changing the `compileSdkVersion` in `ext` ?